xen-ocaml/bindings/page_stubs.c

```diff
@@ -25,6 +25,9 @@
 #include <caml/fail.h>
 #include <caml/bigarray.h>
 
+/* XXX TODO defined in minios */
+void *_xmalloc(size_t size, size_t align);
```

Comment:
I was wondering why MiniOS doesn't just call this `memalign` since it's got exactly the same declaration and would save on an unnecessary divergence from POSIX...